content/maxwell3_fdem/MT/MT_N_layered_Earth.rst

```diff
@@ -0,0 +1,187 @@
+.. _MT_N_layered_Earth:
+
+Response of a layered Earth to a plane wave
+===========================================
+
+This work follows the derivation in :cite:`ward1988` and is supported by interactive apps developed in a `binder`_.
+
+.. _binder: http://mybinder.org/repo/ubcgif/em_examples/notebooks/geophysical_surveys/MT_N_Layered_Earth/MT_n_layered_earth_example.ipynb
+
+
+
+
+Introduction
+------------
+
+ .. figure:: images/MT_N_layered_Earth-1.hires.png
+    :align: center
+    :scale: 20% 
+    :name: MTlayeredEarth
+
+    Earth Model Configuration
+
+The problem setup is shown in the figure below (See :numref:`MTlayeredEarth`), where we have
+
+| -a layered Earth, each layer with its own physical properties \\(\\sigma, \\varepsilon, \\mu \\) 
+
+| -a plane wave coming from air, perpendicular to the Earth's surface, traveling along the axis Z (oriented downward), with an electric field \\(\\mathbf{E_x}\\) and a magnetic field \\(\\mathbf{H_y}\\)
```

Comment:
here, if you remove the `|` and put a space after the `-`, the bullet points will render. ie
```
 - a layered Earth, each layer with its own physical properties :math:`\sigma, \varepsilon, \mu` 
 - a plane wave coming from air, perpendicular to the Earth's surface, traveling along the axis Z (oriented downward), with an electric field \\(\\mathbf{E_x}\\) and a magnetic field \\(\\mathbf{H_y}\\)
 - the origin of coordinate system coincides with the Earth's surface
```